plugins/org.jkiss.dbeaver.ext.altibase/src/org/jkiss/dbeaver/ext/altibase/AltibaseConstants.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jkiss.dbeaver.ext.altibase;
+
+import org.jkiss.utils.StandardConstants;
+
+public class AltibaseConstants {
+    
+    public static final String NEW_LINE = System.getProperty(StandardConstants.ENV_LINE_SEPARATOR);
+    public static final String SPACE4 = "    ";
+    public static final String PSM_POSTFIX = ";" + AltibaseConstants.NEW_LINE + "/";
+
+    public static final String PUBLIC_USER = "PUBLIC";
+    
+    public static final String SYSTEM_GENERATED_PREFIX = "__SYS_";
+    
+    public static final int PACKAGE_TYPE_SPEC = 6;
+    public static final int PACKAGE_TYPE_BODY = 7;
+
+    public static final int TYPE_BIGINT = -5;

```

Comment:
It is more or less copy of java.sql.Types constants:
TYPE_BIGINT == Types.BIGINT
etc
Let's use original constants

Reply:
I absolutely agree with your opinion. Unfortunately, Altibase data types do not perfectly match java.sql.Types. For types compatible with java.sql.Type, original constants were used as much as possible as you suggested.

Thanks for your review :)